ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-intended-config.j2

```diff
@@ -126,6 +126,10 @@
 {% include 'eos/virtual-source-nat.j2' %}
 {# event handler #}
 {% include 'eos/event-handler.j2' %}
+{# BGP Groups configuration #}
```

Comment:
This is more cosmetic, but the ordered generate doesn't match what I tested on 4.25.4M DUT.

```shell
vrf instance MGMT
!
group bgp bar
   vrf red
   neighbor peer-group-baz
   maintenance profile bgp downlink-neighbors
   exit
!
group bgp foo
   neighbor fe80::1
   neighbor 169.254.1.1
   maintenance profile bgp uplink-neighbors
   maintenance profile bgp ixp
   exit
!
interface Port-Channel5
   description MLAG_PEER_CN-IT-BL1B_Po5
   switchport trunk allowed vlan 2-4094
   switchport mode trunk
   switchport trunk group LEAF_PEER_L3
   switchport trunk group MLAG
!
interface Ethernet1
   description P2P_LINK_TO_CN-SPINE1_Ethernet8
   no switchport
   ip address 172.30.254.25/31
!
interface Ethernet2
   description P2P_LINK_TO_CN-SPINE2_Ethernet8
   no switchport
   ip address 172.30.254.27/31
!
interface Ethernet3
   description P2P_LINK_TO_CN-SPINE3_Ethernet8
   no switchport
   ip address 172.30.254.29/31
!
interface Ethernet4
   description P2P_LINK_TO_CN-SPINE4_Ethernet8
   no switchport
   ip address 172.30.254.31/31
!
interface Ethernet5
   description MLAG_PEER_CN-IT-BL1B_Ethernet5
   channel-group 5 mode active
!
interface Ethernet6
   description MLAG_PEER_CN-IT-BL1B_Ethernet6
   channel-group 5 mode active
!
interface Ethernet7
!
interface Ethernet8
!
interface Ethernet9
!
interface Ethernet10
!
interface Ethernet11
!
interface Ethernet12
!
interface Ethernet13
!
interface Ethernet14
!
interface Ethernet15
!
interface Loopback0
   description EVPN_Overlay_Peering
   ip address 192.168.255.8/32
!
interface Loopback1
   description VTEP_VXLAN_Tunnel_Source
   ip address 192.168.254.8/32
!
interface Management1
   description oob_management
   vrf MGMT
   ip address 192.168.200.110/24
!
interface Vlan4093
   description MLAG_PEER_L3_PEERING
   ip address 10.255.251.6/31
!
interface Vlan4094
   description MLAG_PEER
   no autostate
   ip address 10.255.252.6/31
!
interface Vxlan1
   vxlan source-interface Loopback1
   vxlan virtual-router encapsulation mac-address mlag-system-id
   vxlan udp-port 4789
!
group interface QSFP_Interface_Group
   interface Et1,5
   maintenance profile interface uplink-interfaces
   exit
!
group interface SFP_Interface_Group
   maintenance profile interface ix-interfaces
   maintenance profile interface downlink-interfaces
   maintenance profile bgp local-ix
   maintenance profile bgp downlink-neighbors
   exit
!
```